### ISSUE TITLE
Support arbitrary numerals 

### ISSUE THREAD

**Issue by runarberg** [runarberg/ascii2mathml#2]
Currently only [indo-arabic numerals][1] are interpreted as numerals `<mn>`. We should recognize the different [numeral system][2] out there and convert *all* unicode numeral codepoint to an `<mn>` tag.

That however is not enough. People often write numerals with non-numeral characters. For example “42” can be written as “XLII”, “0x2A”, “4.2e+01”, “forty two”, etc. So there needs to be a way to force some delimited text to be interpreted as a numeral.

I recommend the notation `` #`<numeral>` ``. Since the [octothorpe][3] (#) is commonly thought of as “the number sign” and makes initiative sense. The backtick delimitation follows the identifier convention in a way that makes the *#* act like a font command. The only problem is that people who want to use *#* as an operator in front of a forced identifier will be forces to force the *#* as well (`` \#`foo` ``) or separate with whitespace (`` # `foo` ``).

In short this issue asks for two things:
* Interpret unicode numerals as numerals: `四十二` => `<mn>四十二</mn>`.
* Allow forced numerals: `` #`forty two` `` => `<mn>forty two</mn>`.

[1]: https://en.wikipedia.org/wiki/Arabic_numerals
[2]: https://en.wikipedia.org/wiki/Numeral_system
[3]: http://www.shadycharacters.co.uk/2011/05/the-octothorpe-part-1-of-2/
